virttest/utils_hotplug.py

```diff
@@ -458,3 +460,25 @@ def check_vcpu_value(vm, exp_vcpu, vcpupin=None, option="", guest_agent=False):
             final_result = False
 
     return final_result
+
+
+def get_load_per(session=None, iterations=2, interval=10.0):
+    """
+    Get the percentage of load in Guest/Host
+    :param session: Guest Sesssion
+    :param iterations: iterations
+    :param interval: interval between load calculation
+    :return: load of system in percentage
+    """
+    idle_secs = []
+    idle_per = []
+    cmd = "cat /proc/uptime"
```

Comment:
can we reuse `utils_misc.get_uptime()` to retrieve uptime ?

Reply:
@balamuruhans lets not complicate `uptime` function, this method relies on idle value, Second option from that file.